### PR TITLE
Fixes to the ZeroBias DQM sequences

### PR DESCRIPTION
**Pull request by fabiocos**
#### PR description:

This PR is a follow-up of #28325 to fix the crashes after the revision of DQM sequences in #28156 .
There are two changes:

- the DQMMessageLogger sequence for harvesting step is updated;

- the tracker ZeroBias sequence is moved back to the old value.

Both changes need to be validated by @jfernan2

#### PR validation:

workflows 134.71 and 136.856 run, the previously appearing warning on missing sequence in harvesting has disappeared.

### CODE REVIEW

**Comment by fabiocos** [comment]
please test workflow 134.71,136.789,136.801,136.88

**Comment by fabiocos** [comment]
@jfernan2 this PR appears to fix the residual crashes in the IB. I am not sure this is what you intended to get in your revision, please have a look. In case I will integrate them in one of the next IBs and then you may go through a further update.

**Comment by fabiocos** [comment]
@smuzaffar FYI

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Comment by jfernan2** [comment]
+1
Thanks @fabiocos 
I was away for some days, I don't understand how these workflows passed the test in my original PR

**Comment by fabiocos** [comment]
@jfernan2 this was a minimal solution to get this running. Is this that you intend to have finally? This was originally introduced by @mmusich but of course I am not so expert in it to judge whether it is the finally desired sequence (apart that it is what used previously)

**Comment by jfernan2** [comment]
I understand, I am working now on a fix to your fix to have it as desired, I will commit a PR this morning
Thanks

File: DQMOffline/Configuration/python/autoDQM.py [hunk 1]
```diff
@@ -1,18 +1,18 @@
 autoDQM = { 'common': ['@dcs+DQMMessageLoggerSeq+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@castor+@physics+@tau',
                         'PostDQMOffline',
-                        '@dcs+DQMMessageLoggerSeq+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@fed+@tau+dqmFastTimerServiceClient'],
```
Comment (fioriNTU) [inline DQMOffline/Configuration/python/autoDQM.py]: @fabiocos and @jfernan2 wouldn't be easier to define a new key, like:
'DQMMessageLogger' : ['DQMMessageLoggerSeq', 
                                        'PostDQMOffline',
                                        'DQMMessageLoggerClientSeq']
and then call @DQMMessageLogger in both step1 and step2 ?

Comment (jfernan2) [reply DQMOffline/Configuration/python/autoDQM.py]: That was the approach I was testing now... Thanks!

**Comment by mmusich** [comment]
>I don't understand how these workflows passed the test in my original PR

I am not sure the ZB workflows are part if the limited or short matrices (so might have escaped internal scrutiny).
@fabiocos as ZB are a key ingredient of tracker/tracking data certification would it be possible to pick at least one of them in the matrix run for PR tests?

**Comment by fabiocos** [comment]
@mmusich no, these ZB workflows are a bit special and not part of the current short matrix:

```
found a total of  803  workflows:
      of which the following 34 were selected:
4.22   RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC 
4.53   RunPhoton2012B+HLTD+RECODR1reHLT+HARVESTDR1reHLT 
5.1    TTbarFS+HARVESTFS                   
7.3    CosmicsSPLoose_UP18+DIGICOS_UP18+RECOCOS_UP18+ALCACOS_UP18+HARVESTCOS_UP18 
8.0    BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS 
9.0    Higgs200ChargedTaus+DIGI+RECO+HARVEST 
25.0   TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT 
135.4  ZEEFS_13+HARVESTUP15FS+MINIAODMCUP15FS 
136.731 RunSinglePh2016B+HLTDR2_2016+RECODR2_2016reHLT_skimSinglePh_HIPM+HARVESTDR2 
136.7611 RunJetHT2016E_reminiaod+REMINIAOD_data2016_HIPM+HARVESTDR2_REMINIAOD_data2016_HIPM 
136.788 RunSinglePh2017B+HLTDR2_2017+RECODR2_2017reHLT_skimSinglePh_Prompt+HARVEST2017 
136.8311 RunJetHT2017F_reminiaod+REMINIAOD_data2017+HARVEST2017_REMINIAOD_data2017 
136.85 RunEGamma2018A+HLTDR2_2018+RECODR2_2018reHLT_skimEGamma_Offline_L1TEgDQM+HARVEST2018_L1TEgDQM 
140.53 RunHI2011+RECOHID11+HARVESTDHI      
140.56 RunHI2018+RECOHID18+HARVESTDHI18    
158.0  HydjetQ_B12_5020GeV_2018_ppReco+DIGIHI2018PPRECO+RECOHI2018PPRECO+ALCARECOHI2018PPRECO+HARVESTHI2018PPRECO 
1306.0 SingleMuPt1_UP15+DIGIUP15+RECOUP15+HARVESTUP15 
1325.7 TTbar_13_94Xv2NanoAODINPUT+NANOEDMMC2017_94XMiniAODv2+HARVESTNANOAODMC2017_94XMiniAODv2 
1330.0 ZMM_13+DIGIUP15+RECOUP15_L1TMuDQM+HARVESTUP15_L1TMuDQM+NANOUP15 
101.0  SingleElectronE120EHCAL             
25202.0 TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25+NANOUP15_PU25 
1000.0 RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT 
1001.0 RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVDSIPIXELCALRUN1+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4+ALCAHARVD5 
10024.0 TTbar_13TeV_TuneCUETP8M1_2017_GenSimFull+DigiFull_2017+RecoFull_2017+HARVESTFull_2017+ALCAFull_2017+NanoFull_2017 
10042.0 ZMM_13TeV_TuneCUETP8M1_2017_GenSimFull+DigiFull_2017+RecoFull_2017+HARVESTFull_2017+ALCAFull_2017+NanoFull_2017 
10224.0 TTbar_13TeV_TuneCUETP8M1_2017PU_GenSimFull+DigiFullPU_2017PU+RecoFullPU_2017PU+HARVESTFullPU_2017PU+NanoFull_2017PU 
10824.0 TTbar_13TeV_TuneCUETP8M1_2018_GenSimFull+DigiFull_2018+RecoFull_2018+HARVESTFull_2018+ALCAFull_2018+NanoFull_2018 
11634.0 TTbar_14TeV_TuneCUETP8M1_2021_GenSimFull+DigiFull_2021+RecoFull_2021+HARVESTFull_2021+ALCAFull_2021 
12434.0 TTbar_14TeV_TuneCUETP8M1_2023_GenSimFull+DigiFull_2023+RecoFull_2023+HARVESTFull_2023+ALCAFull_2023 
20034.0 TTbar_14TeV_TuneCUETP8M1_2026D35_GenSimHLBeamSpotFull14+DigiFullTrigger_2026D35+RecoFullGlobal_2026D35+HARVESTFullGlobal_2026D35 
20434.0 TTbar_14TeV_TuneCUETP8M1_2026D41_GenSimHLBeamSpotFull14+DigiFullTrigger_2026D41+RecoFullGlobal_2026D41+HARVESTFullGlobal_2026D41 
21234.0 TTbar_14TeV_TuneCUETP8M1_2026D44_GenSimHLBeamSpotFull14+DigiFullTrigger_2026D44+RecoFullGlobal_2026D44+HARVESTFullGlobal_2026D44 
22034.0 TTbar_14TeV_TuneCUETP8M1_2026D46_GenSimHLBeamSpotFull14+DigiFullTrigger_2026D46+RecoFullGlobal_2026D46+HARVESTFullGlobal_2026D46 
250202.181 TTbar_13UP18+PREMIXUP18_PU25+DIGIPRMXLOCALUP18_PU25+RECOPRMXUP18_PU25+HARVESTUP18_PU25 
```

I am not terribly concerned about this, as they are run in any IB, and this is likely the reason why @jfernan2 missed it, unless he did an explicit test. Breaking the IB for a few rounds for such a large refactoring is not something terrible, provided we fix it quickly, and I try to wait the last moment before an important build for this kind of migrations. And if we pick one we might miss an issue in another (there were two distinct categories of problems in this case). 

It is clear that whatever new implementation will have to be explicitly tested against this subset of dedicated workflows.

**Comment by jfernan2** [comment]
I second the proposal by @mmusich . Despite I made runThematrix -i all  it seems I totally missed these failures in the jungle of outputs provided by the command which went beyond the total capacity of my local account. Thanks

**Pull request merged**